code/controllers/subsystem/vote.dm

```diff
@@ -224,7 +236,7 @@ SUBSYSTEM_DEF(vote)
 				var/list/maps = list()
 				for(var/map in global.config.maplist)
 					var/datum/map_config/VM = config.maplist[map]
-					if(!VM.is_votable())
+					if(!VM.is_votable() || SSmapping.config.map_name == VM.map_name) //Always rotate away from current map
```

Comment:
Enforcing variety isn't the worst decision given we do have overlapping map pop bounds.

Reply:
@Bokkiewokkie What are your thoughts? Should we prevent them from getting the same map twice in a row?

Reply:
I don't think constantly rotating is ideal because of how much our pop fluctuates, sometimes. Sometimes you want to play Galactica twice in a row because that one evening has a lot of players but this would mean they can't. Also things like playing Eclipse twice on a row on lower pops because it's more suitable very few players